package.json

```diff
@@ -3,7 +3,7 @@
     "publisher": "Wonderstorm",
     "displayName": "Fracas",
     "description": "Syntax highlighting, code navigation, document formatting, and REPL support for the Fracas and Racket programming languages",
-    "version": "0.1.8",
+    "version": "0.1.9",
```

Comment:
@thisisthedave I noticed we already have 0.1.9 checked into `tdp1` - is this repo's `master` branch up-to-date? I wasn't sure if `master` contains all your fixes for generating localized text C++, etc.

Reply:
`master` didn't have my fixes until a few minutes ago because I never got a code review. I merged it anyway just now.